dipy/utils/arrfuncs.py

```diff
@@ -28,3 +28,45 @@ def as_native_array(arr):
         return arr
     return arr.byteswap().newbyteorder()
 
+
+def pinv_vec(a, rcond=1e-15):
+    """Vectorized version of numpy.linalg.pinv"""
+
+    a = np.asarray(a)
+    if np.version.version.split('.') < ['1', '8']:
```

Comment:
Use `LooseVersion` to compare versions. Like here: https://github.com/nipy/dipy/blob/master/dipy/core/optimize.py#L14